src/components/NavigatorBar.js

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import PropTypes from "prop-types"
+
+const NavigatorBar = (props) => {
+  let counter = 0
+  const navLinks = props.menuList.map((menu) => {
+    return (
+      <a href={menu.route} key={counter++}>

```

Comment:
you might wanna use `index` of array instead of `counter`

```
const navLinks = props.menuList.map(({ route, name }, index) => (
  <a href={route} key={index}>
    <li>{name}</li>
  </a>
)
```

Reply:
and could also destructure `props` since it has only one key
and then move `navLinks` into line 15